canonical-websites/www.ubuntu.com
# 2660: Move /about/canonical-and-ubuntu to /community/canonical

# Note
**Please merge #2685 first**

## Done

- Moved /about/canonical-and-ubuntu to /community/canonical
- Updated copy according to [copydoc](https://docs.google.com/document/d/1kup0vH-dschLdqhjp7CuktEa8CNoLPSm5Fo24y4y1lc/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/community/canonical
- Check that it matches the [copydoc](https://docs.google.com/document/d/1kup0vH-dschLdqhjp7CuktEa8CNoLPSm5Fo24y4y1lc/edit#)
- Check that /about/canonical-and-ubuntu redirects to /community/canonical

## Issue / Card

Fixes #2660
